File: app/src/main/java/id/ac/ui/cs/mobileprogramming/irfanazizalamin/helloworld/MainActivity.java
package id.ac.ui.cs.mobileprogramming.irfanazizalamin.helloworld;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    private TaskListFragment taskListFragment = new TaskListFragment();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportFragmentManager().beginTransaction()
                .add(R.id.container, taskListFragment)
                .addToBackStack("task_list")
                .commit();
    }
}

File: app/src/main/java/id/ac/ui/cs/mobileprogramming/irfanazizalamin/helloworld/MainActivity.kt
package id.ac.ui.cs.mobileprogramming.irfanazizalamin.helloworld

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.google.android.material.floatingactionbutton.FloatingActionButton
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var count = 0
        var counterText: TextView = findViewById(R.id.counterText)

        findViewById<FloatingActionButton>(R.id.fab).setOnClickListener { view ->
            count += 1
            counterText.setText(count.toString());
        }
    }
}